backend/src/org/commcare/xml/FixtureXmlParser.java

```diff
@@ -50,11 +50,14 @@ public FormInstance parse() throws InvalidStructureException, IOException,
 
         String userId = parser.getAttributeValue(null, "user_id");
 
-        //Get to the data root
-        parser.nextTag();
+        TreeElement root;
+        if (nextTagInBlock("fixture")) {
+            //TODO: We need to overwrite any matching records here.
+            root = new TreeElementParser(parser, 0, fixtureId).parse();
+        } else {
+            root = new TreeElement();
```

Comment:
Cool, updated such that `parse()` returns `null` for empty fixtures. Do you forsee this breaking anything?